addons/pos_loyalty/static/src/js/Loyalty.js

```diff
@@ -855,7 +862,7 @@ const PosLoyaltyOrder = (Order) => class PosLoyaltyOrder extends Order {
      */
     pointsForPrograms(programs) {
         pointsForProgramsCountedRules = {};
-        const orderLines = this.get_orderlines();
+        const orderLines = this._get_non_refund_order_lines();
```

Comment:
```suggestion
        const orderLines = this.get_orderlines().filter((line) => !line.refunded_orderline_id);
```
are you ok with this change ?

Reply:
Yes, just amended now. Will see how tests do